editors/code/src/debug.ts

```diff
@@ -90,9 +96,12 @@ async function getDebugConfiguration(
     }
 
     if (!debugEngine) {
+        const commandCodeLLDB: string = createCommandLink("vadimcn.vscode-lldb");
+        const commandCpp: string = createCommandLink("ms-vscode.cpptools");
+
         await vscode.window.showErrorMessage(
-            `Install [CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb)` +
-                ` or [MS C++ tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) extension for debugging.`,
+            `Install [CodeLLDB](command:${commandCodeLLDB} "Install CodeLLDB")` +
```

Comment:
```suggestion
            `Install [CodeLLDB](command:${commandCodeLLDB} "Open CodeLLDB")` +
            ` or the [C/C++](command:${commandCpp} "Open C/C++") extension for debugging.`,
```

Reply:
I accepted your suggestion but unfortunately it had an error - a line was duplicated in your suggestion, I'll push the patch manually.

Reply:
Yeah, multi-line suggestions don't (always?) work.